src/neoxp/Commands/CreateCommand.cs

```diff
@@ -29,7 +29,7 @@ public CreateCommand(ExpressChainManagerFactory chainManagerFactory, IFileSystem
             this.txExecutorFactory = txExecutorFactory;
         }
 
-        [Argument(0, Description = $"Name of {EXPRESS_EXTENSION} file to create\nDefault location is home directory as:\nLinux: $HOME/.neo-express/{DEFAULT_EXPRESS_FILENAME}\nWindows: %UserProfile%\\.neo-express\\{DEFAULT_EXPRESS_FILENAME}")]
+        [Option(Description = $"Name of {EXPRESS_EXTENSION} file to create\nDefault location is home directory as:\nLinux: $HOME/.neo-express/{DEFAULT_EXPRESS_FILENAME}\nWindows: %UserProfile%\\.neo-express\\{DEFAULT_EXPRESS_FILENAME}")]
```

Comment:
Maybe we should keep this as argument and add new `--global` for adding to global folder?